diff --git a/node_modules/panzoom/index.d.ts b/node_modules/panzoom/index.d.ts
index 84f20a5..6d16aa3 100644
--- a/node_modules/panzoom/index.d.ts
+++ b/node_modules/panzoom/index.d.ts
@@ -36,6 +36,8 @@ declare module "panzoom" {
     pinchSpeed?: number;
     beforeWheel?: (e: WheelEvent) => void;
     beforeMouseDown?: (e: MouseEvent) => void;
+    beforeDoubleClick?: (e: MouseEvent) => void;
+    beforeTouch?: (e: TouchEvent) => void;
     autocenter?: boolean;
     onTouch?: (e: TouchEvent) => void;
     onDoubleClick?: (e: Event) => void;
diff --git a/node_modules/panzoom/index.js b/node_modules/panzoom/index.js
index e4c59b2..76778da 100644
--- a/node_modules/panzoom/index.js
+++ b/node_modules/panzoom/index.js
@@ -66,6 +66,8 @@ function createPanZoom(domElement, options) {
   var zoomDoubleClickSpeed = typeof options.zoomDoubleClickSpeed === 'number' ? options.zoomDoubleClickSpeed : defaultDoubleTapZoomSpeed;
   var beforeWheel = options.beforeWheel || noop;
   var beforeMouseDown = options.beforeMouseDown || noop;
+  var beforeDoubleClick = options.beforeDoubleClick || noop;
+  var beforeTouch = options.beforeTouch || noop;
   var speed = typeof options.zoomSpeed === 'number' ? options.zoomSpeed : defaultZoomSpeed;
   var transformOrigin = parseTransformOrigin(options.transformOrigin);
   var textSelection = options.enableTextSelection ? fakeTextSelectorInterceptor : domTextSelectionInterceptor;
@@ -501,6 +503,7 @@ function createPanZoom(domElement, options) {
     }
 
     smoothScroll.cancel();
+    cancelZoomAnimation();
 
     releaseDocumentMouse();
     releaseTouches();
@@ -579,8 +582,15 @@ function createPanZoom(domElement, options) {
   }
 
   function onTouch(e) {
-    // let the override the touch behavior
-    beforeTouch(e);
+    if (beforeTouch(e)) {
+      return;
+    }
+
+    const hasTouchOption = ('onTouch' in options);
+    if (!hasTouchOption || (hasTouchOption && options.onTouch(e))) {
+      e.preventDefault();
+      e.stopPropagation();
+    }
 
     if (e.touches.length === 1) {
       return handleSingleFingerTouch(e, e.touches[0]);
@@ -604,18 +614,6 @@ function createPanZoom(domElement, options) {
     e.preventDefault();
   }
 
-  function beforeDoubleClick(e) {
-    // TODO: Need to unify this filtering names. E.g. use `beforeDoubleClick``
-    if (options.onDoubleClick && !options.onDoubleClick(e)) {
-      // if they return `false` from onTouch, we don't want to stop
-      // events propagation. Fixes https://github.com/anvaka/panzoom/issues/46
-      return;
-    }
-
-    e.preventDefault();
-    e.stopPropagation();
-  }
-
   function handleSingleFingerTouch(e) {
     var touch = e.touches[0];
     var offset = getOffsetXY(touch);
@@ -719,7 +717,16 @@ function createPanZoom(domElement, options) {
   }
 
   function onDoubleClick(e) {
-    beforeDoubleClick(e);
+    if (beforeDoubleClick(e)) {
+      return;
+    }
+
+    const hasDoubleClickOption = ('onDoubleClick' in options);
+    if (!hasDoubleClickOption || (hasDoubleClickOption && options.onDoubleClick(e))) {
+      e.preventDefault();
+      e.stopPropagation();
+    }
+
     var offset = getOffsetXY(e);
     if (transformOrigin) {
       // TODO: looks like this is duplicated in the file.
@@ -1048,4 +1055,3 @@ function autoRun() {
 }
 
 autoRun();
-	
\ No newline at end of file
