diff --git a/dist/index.mjs b/dist/index.mjs
index e3bb2dec4e92b11b8886954e92bef19f4a8219c4..88d40d4fac4bd9c83c0e28d5bc80bbcb6f6ce4ab 100644
--- a/dist/index.mjs
+++ b/dist/index.mjs
@@ -1009,10 +1009,11 @@ function createDtsResolvePlugin({ cwd, tsconfig, tsconfigRaw, resolver, sideEffe
 					external: true,
 					moduleSideEffects: sideEffects
 				};
-				if (RE_CSS.test(id)) {
-					debug$1("Externalizing css import:", id);
-					return external;
-				}
+				// PATCHED: Disabled CSS externalization
+				// if (RE_CSS.test(id)) {
+				// 	debug$1("Externalizing css import:", id);
+				// 	return external;
+				// }
 				const rolldownResolution = await this.resolve(id, importer, options);
 				debug$1("Rolldown resolution for dts import %O from %O: %O", id, importer, rolldownResolution);
 				if (rolldownResolution?.external) {
