diff --git a/node_modules/sku/config/babel/babelConfig.js b/node_modules/sku/config/babel/babelConfig.js
index f61c7c2..46975ee 100644
--- a/node_modules/sku/config/babel/babelConfig.js
+++ b/node_modules/sku/config/babel/babelConfig.js
@@ -45,6 +45,7 @@ module.exports = ({
     require.resolve('babel-plugin-macros'),
     require.resolve('@loadable/babel-plugin'),
     [require.resolve('babel-plugin-treat'), { alias: 'sku/treat' }],
+    require.resolve('@mattsjones/css-babel-plugin')
   ];
 
   if (hot && isBrowser) {
diff --git a/node_modules/sku/config/webpack/plugins/sku-webpack-plugin/index.js b/node_modules/sku/config/webpack/plugins/sku-webpack-plugin/index.js
index 559e25a..42e33a7 100644
--- a/node_modules/sku/config/webpack/plugins/sku-webpack-plugin/index.js
+++ b/node_modules/sku/config/webpack/plugins/sku-webpack-plugin/index.js
@@ -12,6 +12,7 @@ const {
   SVG,
   DEPRECATED_CSS_IN_JS,
   resolvePackage,
+  makeVanillaCssLoaders,
 } = require('../../utils');
 const createTreatPlugin = require('../createTreatPlugin');
 const defaultCompilePackages = require('../../../../context/defaultCompilePackages');
@@ -64,7 +65,7 @@ class SkuWebpackPlugin {
         include: this.include,
         oneOf: [
           {
-            compiler: 'treat-webpack-loader',
+            compiler: /^treat-webpack/,
             use: makeJsLoaders({
               target: 'node',
               lang: 'ts',
@@ -148,6 +149,15 @@ class SkuWebpackPlugin {
             }),
           }),
       },
+      {
+        test: /\.vanilla.css$/i,
+        use: makeVanillaCssLoaders({
+          isProductionBuild,
+          MiniCssExtractPlugin,
+          hot,
+          supportedBrowsers,
+        })
+      },
       {
         test: IMAGE,
         include: this.include,
diff --git a/node_modules/sku/config/webpack/utils/loaders.js b/node_modules/sku/config/webpack/utils/loaders.js
index 2160931..ab1a813 100644
--- a/node_modules/sku/config/webpack/utils/loaders.js
+++ b/node_modules/sku/config/webpack/utils/loaders.js
@@ -89,6 +89,37 @@ const makeCssLoaders = (options = {}) => {
   ];
 };
 
+const makeVanillaCssLoaders = (options = {}) => {
+  const {
+    isProductionBuild,
+    MiniCssExtractPlugin,
+    supportedBrowsers,
+    hot = false,
+  } = options;
+
+  return [
+    {
+      loader: MiniCssExtractPlugin.loader,
+      options: {
+        hmr: hot,
+      },
+    },
+    {
+      loader: require.resolve('css-loader'),
+    },
+    {
+      loader: require.resolve('postcss-loader'),
+      options: {
+        plugins: () => [
+          require('autoprefixer')(supportedBrowsers),
+          // Minimize CSS on production builds
+          ...(isProductionBuild ? [require('cssnano')()] : []),
+        ],
+      },
+    },
+  ];
+};
+
 const makeImageLoaders = (options = {}) => {
   const { target = 'browser' } = options;
 
@@ -130,6 +161,7 @@ const makeSvgLoaders = () => [
 module.exports = {
   makeJsLoaders,
   makeCssLoaders,
+  makeVanillaCssLoaders,
   makeImageLoaders,
   makeSvgLoaders,
 };
