diff --git a/node_modules/sku/config/jest/jestConfig.js b/node_modules/sku/config/jest/jestConfig.js
index 0e07700..8038ed4 100644
--- a/node_modules/sku/config/jest/jestConfig.js
+++ b/node_modules/sku/config/jest/jestConfig.js
@@ -1,7 +1,7 @@
 const escapeRegex = require('escape-string-regexp');
 const { paths } = require('../../context');
 const slash = '[/\\\\]'; // Cross-platform path delimiter regex
-const compilePackagesRegex = paths.compilePackages.map(escapeRegex).join('|');
+const compilePackagesRegex = paths.compilePackages.map(cp => `.*${escapeRegex(cp)}`).join('|');
 
 module.exports = {
   testEnvironment: 'jsdom',
