diff --git a/yoga-layout/build/Release/nbind.js b/yoga-layout/build/Release/nbind.js
index 0853309f9e0bd49d3e585db266442210d9538d1e..3641becd9a2f5b43b315f99c104f0de593301616 100644
--- a/yoga-layout/build/Release/nbind.js
+++ b/yoga-layout/build/Release/nbind.js
@@ -1143,7 +1143,7 @@
       }
     }_nbind.addMethod = addMethod;function throwError(message) {
       throw new Error(message);
-    }_nbind.throwError = throwError;_nbind.bigEndian = false;_a = _typeModule(_typeModule), _nbind.Type = _a.Type, _nbind.makeType = _a.makeType, _nbind.getComplexType = _a.getComplexType, _nbind.structureList = _a.structureList;var BindType = function (_super) {
+    }_nbind.throwError = throwError;_nbind.bigEndian = false;var _a = _typeModule(_typeModule); _nbind.Type = _a.Type, _nbind.makeType = _a.makeType, _nbind.getComplexType = _a.getComplexType, _nbind.structureList = _a.structureList;var BindType = function (_super) {
       __extends(BindType, _super);function BindType() {
         var _this = _super !== null && _super.apply(this, arguments) || this;_this.heap = HEAPU32;_this.ptrSize = 4;return _this;
       }BindType.prototype.needsWireRead = function (policyTbl) {